src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothGattImpl.cs

```diff
@@ -115,20 +118,37 @@ internal void SendNotification(BluetoothGattCharacteristic characteristic, strin
 
         internal Task<bool> SendIndicationAsync(BluetoothGattServer server, BluetoothGattCharacteristic characteristic, string clientAddress)
         {
-            TaskCompletionSource<bool> tcs = new TaskCompletionSource<bool>();
-            Interop.Bluetooth.BtGattServerNotificationSentCallback cb = (result, address, serverHandle, characteristicHandle, completed, userData) =>
+            TaskCompletionSource<bool> task = new TaskCompletionSource<bool>();
+            int requestId = 0;
+
+            lock (this)
+            {
+                requestId = _requestId++;
+                _sendIndicationTaskSource[requestId] = task;
+            }
+
+            _sendIndicationCallback = (result, address, serverHandle, characteristicHandle, completed, userData) =>
```

Comment:
`_sendIndicationCallback` is overwritten by following request

Reply:
Please refer this PR https://github.com/Samsung/TizenFX/pull/1674
it is same case

Reply:
Done.